Sharpmake/PackageReferences.cs

```diff
@@ -158,10 +158,12 @@ public enum AssetsDependency : uint
             None = 0,
             Compile = 1 << 0,
             Runtime = 1 << 1,
-            ContentFiles = 1 << 2,
+            [Obsolete]
```

Comment:
thanks for the update, but could you add the message argument to the Obsolete('message'), indicating that they should use ContentFiles instead?